docs/extensions/extensions.md

```diff
@@ -0,0 +1,27 @@
+---
+sidebar_position: 8
+---
+
+# 🔌 Extensions
+
+This guide introduces extensions for Scaffold-ETH 2 and explains how to use and create extensions.
+
+## What are Extensions?
+
+Extensions are modular add-ons for Scaffold-ETH 2 that allow developers to enhance and customize their dApps easily. They offer several benefits:
+
+- Seamless integration with the base Scaffold-ETH 2 project
+- Quick addition of new features, pages, contracts, or components
+- Compatibility with Scaffold-ETH 2 core updates and improvements
+
+> Video Guide: For a visual walkthrough of the extension development process, check out our [YouTube tutorial](https://youtu.be/XQCv533XGZk?si=dlJH4zd4b99_6soW).
```

Comment:
Umm I think we could remove this from here? Since I feel maybe in initial intro pitch people would be more interested in how to use extensions rather than creating extensions?

Reply:
yeah true, done!